0.preprocess-sites/7.visualize-cell-summary.py

```diff
@@ -0,0 +1,249 @@
+import os
+import sys
+import pathlib
+import argparse
+import pandas as pd
+import plotnine as gg
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+sys.path.append(os.path.join("..", "scripts"))
+from config_utils import process_config_file
+from cell_quality_utils import CellQuality
+
+config_file = "site_processing_config_CP151A1.yaml"
+force = True
+config = process_config_file(config_file)
+
+#Defines the sections of the config file
+core_args = config["core"]
+spot_args = config["process-spots"]
+cell_args = config["process-cells"]
+summ_cell_args = config["summarize-cells"]
+
+#Defines the variables set in the config file
+batch = core_args["batch"]
+quality_func = core_args["categorize_cell_quality"]
+
+barcode_cols = spot_args["barcode_cols"]
+barcode_cols = ["Metadata_Foci_" + col for col in barcode_cols]
```

Comment:
we don't want to append a prefix if it is already there. We had code somewhere to do this check, right? Might be worth splitting out an `add_prefix()` function at some point.

Reply:
In 2.process-cells, Metadata_Foci_ is added to the beginning of all columns in the foci_df before it is merged into the compartment csvs. For this step, we filter down the columns that are in the df to just a subset. But when we set the column names in the config it is before they are renamed with Metadata_Foci_.